workspaces/orchestrator/README.md

```diff
@@ -171,115 +109,105 @@ yarn dev
    );
    ```
 
-### Extensible Workflow Execution Form
+## Install as a dynamic plugin in Red Hat Developer Hub
+
+Follow [these guidelines](https://github.com/rhdhorchestrator/orchestrator-helm-operator/blob/main/docs/release-1.3/existing-rhdh.md#install-the-orchestrator-operator) to install the orchestrator operator and configure the orchestrator as a dynamic plugin in Red Hat Developer Hub.
+
+## Configuration
 
-The `orchestrator` plugin includes an extensible form for executing forms. For detailed guidance see the [Extensible Workflow Execution Form Documentation](https://github.com/redhat-developer/rhdh-plugins/blob/main/workspaces/orchestrator/plugins/orchestrator/docs/extensibleForm.md).
+### Devmode local configuration
 
-### Setting up permissions
+This configuration serves for running the application locally for development purposes.
 
-The HTTP endpoints exposed by the `orchestrator-backend` can enforce authorization if the [RBAC plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/rbac/plugins) is deployed.
-Please refer the RBAC plugin documentation for the setup steps (mind they rely on the [Backstage authentication and identity](https://backstage.io/docs/auth)).
+#### Prerequisites
 
-More detailed info about permissions can be found in [docs/Permissions.md](docs/Permissions.md).
+- Docker up and running
```

Comment:
Isn't podman supported as well? if not, should we provide here the command for running the container using podman from CLI instead of auto-start by the plugin?

Reply:
the command is invoked in our code, currently it runs docker
we will need to change the code to support also podman

Reply:
I think better change the code to support both than to have the README explain what to do for podman, which is updating the autoStart to false and running the command

Reply:
Right, so far docker is hard coded. It's for devmode only, let's improve for podman in a separate PR